src/assets/styles/10-settings/_colours.scss

```diff
@@ -58,6 +58,8 @@
   --X-colour-error-red: #d2373f;
   --X-colour-error-pink: #ffebeb;
   --X-colour-dark-blue: #003667;
+
+  --hack-safari-transparent-white: rgba(255, 255, 255, 0.01);
```

Comment:
What's this for?

Reply:
In safari background linear gradients which fade to transparent [display very strangely](https://stackoverflow.com/questions/11829410/css3-gradient-rendering-issues-from-transparent-to-white) - the example specified alpha as 0.001 but 0.01 worked better in my case.

Reply:
I'll add a code comment to explain that

Reply:
ace, thanks for that